### PR TITLE
Reader: add Tracks and Google Analytics stats to Reader sidebar

### PR DESCRIPTION
**Pull request by aaronyan**
Adds stats for whenever a user clicks on an item in the Reader sidebar.

### CODE REVIEW

**Comment by aaronyan** [comment]
meta undefined

Currently getting an error where we have an undefined `meta` when calling `recordAction()` in [client/state/console-dispatch/index.js](https://github.com/Automattic/wp-calypso/blob/master/client/state/console-dispatch/index.js#L59).

This only occurs when calling the `recordTrack()` function from client/reader/sidebar/index.jsx. I don't see this error occur when recording sidebar clicks from a tag item or list item.

Any ideas?

**Comment by aaronyan** [comment]
Resolved the above error! :)

File: client/reader/sidebar/index.jsx [hunk 1]
```diff
@@ -104,6 +105,37 @@ export const ReaderSidebar = createReactClass( {
 		}
 	},
 
+	handleReaderSidebarFollowedSitesClicked() {
+		recordAction( 'clicked_reader_sidebar_followed_sites' );
+		recordGaEvent( 'Clicked Reader Sidebar Followed Sites' );
+		return recordTrack( 'calypso_reader_sidebar_followed_sites_clicked', {} );
```
Comment (samouri) [inline client/reader/sidebar/index.jsx]: any reason this is returned?

File: client/reader/stats.js [hunk 2]
```diff
@@ -98,11 +98,6 @@ export function recordTrack( eventName, eventProperties ) {
 		eventProperties = Object.assign( { subscription_count: subCount }, eventProperties );
 	}
 
-	if ( location === 'topic_page' && ! eventProperties.hasOwnProperty( 'tag' ) ) {
```
Comment (samouri) [inline client/reader/stats.js]: whats this about?

**Comment by samouri** [comment]
can you also add a stat for "Manage" clicks?

**Comment by samouri** [comment]
tested out well! 👍 

this will be interesting to examine later

**Comment by samouri** [comment]
https://github.com/Automattic/wp-calypso/pull/16809#discussion_r130951037

File: client/reader/stats.js (hunk 2 above)
Comment (aaronyan) [inline client/reader/stats.js]: Hmm, not sure why I added this [in the first place](https://github.com/Automattic/wp-calypso/commit/863e38908b87ed26dc910127c532a4ef65d67a3c). I can just add the `tag` property when calling `recordTrack()`, right?

File: client/reader/sidebar/index.jsx (hunk 1 above)
Comment (aaronyan) [inline client/reader/sidebar/index.jsx]: no good reason. Removing!

**Comment by samouri** [comment]
LGTM

File: client/reader/sidebar/reader-sidebar-teams/list-item.jsx [hunk 3]
```diff
@@ -8,6 +8,15 @@ import { localize } from 'i18n-calypso';
  * Internal Dependencies
  */
 import ReaderSidebarHelper from '../helper';
+import { recordAction, recordGaEvent, recordTrack } from 'reader/stats';
+
+const handleReaderSidebarTeamsListItemClicked = team => () => {
```
Comment (samouri) [inline client/reader/sidebar/reader-sidebar-teams/list-item.jsx]: fancy fancy syntax ;)

Comment (aaronyan) [inline client/reader/sidebar/reader-sidebar-teams/list-item.jsx]: lol :)

**Pull request merged**